tests/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ else()
 endif()
 
 string(REGEX MATCH
-  "([0-9])\\.([0-9])\\.([0-9])" GTEST_VERSION_PARSED
+  "([0-9])\\.([0-9]+)\\.([0-9])" GTEST_VERSION_PARSED
```

Comment:
Ah, good catch! Thanks.

This should be the only necessary component of this PR; the code in `gmock_fixes.h` is only built for GMock < 1.8.1, so once the version detection is fixed everything should work!